chia/cmds/start_funcs.py

```diff
@@ -31,7 +31,9 @@ def launch_start_daemon(root_path: Path) -> subprocess.Popen:
     return process
 
 
-async def create_start_daemon_connection(root_path: Path, config: Dict[str, Any]) -> Optional[DaemonProxy]:
+async def create_start_daemon_connection(
+    root_path: Path, config: Dict[str, Any], skip_keyring: bool
```

Comment:
```suggestion
    root_path: Path, config: Dict[str, Any], *, skip_keyring: bool
```

I think this would be good too

Reply:
I adopted this suggestion.